Documentation/markdown/SoftwareArchitecture.md

```diff
@@ -15,63 +16,119 @@
 
 ## <a name="Overall_architecture"></a> Overall Software Architecture
 
-This section will give a general overview of the software architecture of the Open Infra Platform with its core software modules together with their interrelations.
+This section gives a general overview of the software architecture of the Open Infra Platform 
+with its core software modules together with their interrelations.
 Furthermore, dependencies on third-party libraries are stated. 
 
 ![](../images/software_architecture.PNG)
 
 ### <a name="Fundaments"></a> Fundaments
-The base module constitutes the backbone of the Open Infra Platform, with all of the other software parts building upon it. As its main part, several third-party libraries are linked here. Other than that, it also provides very fundamental elements like the IModel interface that is implemented by the upper layers for every supported file format (e.g., IFC and OFF) and will hold all of the information read from the input files. Furthermore, classes and structures for georeferencing metadata and simple bounding boxes are given.
-
-Linked third-party libraries:
-* Boost (library for linear algebra, multithreading, and storage management; header-only)
-* BlueFramework (open-source C++ 3D graphics rendering engine developed by TUM)
-* Carve (constructive solid geometry library)
-* Tclap (small and flexible library providing an interface for accessing and defining command line arguments; header-only)
-* FFTW3 (computation of Fourier transformations; header-only)
-* Doxygen (tool to generate documentation; header-only)
+The base module constitutes the backbone of the Open Infra Platform, 
+with all of the other software parts building upon it. 
+As its main part, several third-party libraries are linked here. 
+Other than that, it also provides very fundamental elements like the _IModel_ interface 
+that is implemented by the upper layers for every supported file format (e.g., `IFC` and `OFF`)
+and holds all of the information read from the input files. 
```

Comment:
```suggestion
and provides some basic information about the input files. 
```

It doesn't hold all the information. As a matter of fact, it holds no information. It only *promises*  that some information will be made availlable by the implementation.

Reply:
Resolved: 7aaecb6